.travis.yml

```diff
@@ -1,10 +1,13 @@
 language: php
+
 php:
-- 5.5
-- 5.6
-- hhvm
+  - 5.6
+  - 7
+
 before_script:
-- composer self-update
-- composer install --prefer-source --no-interaction --dev
+  - pecl install mongodb
```

Comment:
mongodb is an optional dependancy. It should not be installed in the test suite (it does not work with HHVM)